### DISCUSSION TITLE
Strange versioning in  1.2.3-1

### DISCUSSION THREAD

**Discussion by SignFinder** [rustdesk/rustdesk#7628] [General]
### Bug Description

Hi team,
Couldyou kindly replace dash to dota in your versions numbers like  1.2.3-1.
Microsoft Package Manager do not understand that  1.2.3-1 is higher than  1.2.3 and asking to update  1.2.3-1 to  1.2.3.

### How to Reproduce

1. Install Rustdesk  1.2.3-1.
2. Use command winget update
3. See that there is suggestion to "update" Rustdesk  1.2.3-1 to 1.2.3.

### Expected Behavior

Winget should understand that version 1.2.3.1 is higher than  1.2.3.

### Operating system(s) on local side and remote side

Windows 11 23H2

### RustDesk Version(s) on local side and remote side

1.2.3-1 - 1.2.3

### Screenshots

![rustdesk_incorrect_versioning](https://github.com/rustdesk/rustdesk/assets/299814/793650bf-2cbd-4509-bb16-94bfa1cde67b)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/22da14e3f70e14ab7c87db483d4470b7f8ffdeef

**Reply by rustdesk** [reply]
No idea who is maintaining the winget source of RustDesk.

**Reply by SignFinder** [reply]
Thanks for your participation and decision to change dash to dota. Ask winget team to learn it to understand dash in versions - might could be such a hard work.
